src/components/ErrorHandling/Sentry.tsx

```diff
@@ -39,10 +40,16 @@ const initializeSentry = () => {
 
 const isInitialized = () => !!getCurrentHub().getClient();
 
+type ErrorMetadata = {
+  gqlErr?: ErrorResponse["graphQLErrors"][0];
+  operationName?: ErrorResponse["operation"]["operationName"];
+  variables?: ErrorResponse["operation"]["variables"];
```

Comment:
These are all actually typed from
`import { GraphQLError } from "graphql";` since they are properties of the GraphQL request and not specific to Apollo.

Reply:
This is the typing of Apollo's onError handlers if I'm not mistaken: https://github.com/apollographql/apollo-client/blob/17ccc42825d0b0355212f3ac823780edb467ea77/src/link/error/index.ts#L28